int8 UTC_GPS_time
bool bogie_connection_1
bool bogie_connection_2
bool bogie_connection_3
bool arm_connection_status
bool arm_end_effector_connection_statuses
bool camera_zed
bool camera_undercarriage
bool camera_chassis
bool camera_main_navigation
bool sample_containment_connection_status
bool tower_connection_status
bool chassis_pan_tilt_connection_status
int8 GPS_connection_status
float64 jetson_CPU
float64 jetson_RAM
float64 jetson_EMMC
float64 jetson_NVME_SSD
float64 jetson_GPU_temp
bool FrSky_controller_connection_status